packages/suite-web-landing/pages/_document.tsx

```diff
@@ -6,6 +6,7 @@ import globalStyles from '../support/styles';
 import { isEnabled } from '@suite-utils/features';
 
 const isOnionLocation = isEnabled('ONION_LOCATION_META');
+const SUITE_URL = 'https://suite.trezor.io';
```

Comment:
Very minor but this could be exported from `packages/suite/utils/urls.ts`.

Reply:
I'd say should. But the thruth is that my idea of having all urls in one file was to be able to easily test if they are still alive, which is not very relevant for suite url.

Reply:
Addressed in b7c2c2fbd916379c016c7086ba7b5d38d3af59c8